pkg/keyspace/tso_keyspace_group.go

```diff
@@ -590,14 +625,19 @@ func (m *GroupManager) FinishSplitKeyspaceByID(splitTargetID uint32) error {
 
 // GetNodesNum returns the number of nodes.
 func (m *GroupManager) GetNodesNum() int {
```

Comment:
```suggestion
func (m *GroupManager) GetNodesCount() int {
```
Use count instead of number will be clearer.